content/docs/tech-resources/using-proxy-https-tracking-domain.md

```diff
@@ -1,5 +1,5 @@
 ---
-lastUpdated: "04/13/2021"
+lastUpdated: "12/23/2021"
```

Comment:
Should we change the last updated timestamp?  We didn't review and update the content just fixed a bug.

Reply:
Oh I see .. I thought I was meant to touch it anyway. Can revert.